backend/packages/wps-weather/README.md

```diff
@@ -0,0 +1,68 @@
+# wps-weather
+
+Weather maps and meteorological data processing for Wildfire Predictive Services Unit.
+
+Contains utilities for fetching, processing, and visualizing weather data including maps, forecasts, and meteorological datasets.
+
+## Standalone installation with uv
+
+### Installing uv
+
+First, install uv if you don't have it:
+
+**macOS/Linux:**
+```bash
+curl -LsSf https://astral.sh/uv/install.sh | sh
+```
+
+**Windows:**
+```powershell
+powershell -c "irm https://astral.sh/uv/install.ps1 | iex"
+```
+
```

Comment:
Could also add the homebrew option:
`HOMEBREW_NO_AUTO_UPDATE=1 brew install uv`

Reply:
Done in https://github.com/bcgov/wps/pull/4968/commits/24c9d38a75a2a9e2478e781190095878b4cf66a7